.ci/dockerfiles/Dockerfile.rocm

```
@@ -0,0 +1,608 @@
+FROM builder-base AS final
+
+ARG _UID=148069
+ARG _GID=30
+ARG _LOGIN=svc-nixl
+ARG _GROUP=hardware
+ARG _HOME=/home/$_LOGIN
+ARG ARCH=x86_64
+ARG NIXL_INSTALL_DIR=/opt/nixl
+
+# User/group + sudo setup for the non-root service account.
+RUN if ! getent group "${_GID}" > /dev/null 2>&1; then \
```

Hadolint (2.14.0)

[warning] `useradd` without flag `-l` and high UID will result in excessively large Image.

(DL3046)